contracts/src/RealitioForeignArbitrationProxyWithAppeals.sol

```diff
@@ -0,0 +1,602 @@
+     */
+    function initialize(string calldata _metaEvidence, string calldata _termsOfService) external onlyGovernor {
+        require(!initialized, "Proxy already initialized");
+        require(homeProxy != address(0), "Home proxy not set");
+
+        initialized = true;
+        termsOfService = _termsOfService;
+        emit MetaEvidence(0, _metaEvidence);
+    }
+
+    /**
+     * @notice Changes the address of a governor.
+     * @param _governor The address of the new governor.
+     */
+    function changeGovernor(address _governor) external onlyGovernor {
+        governor = _governor;
+    }
+
+    /**
+     * @notice Sets the address of the arbitration proxy on the Home Chain.
+     * @param _homeProxy The address of the proxy.
+     * @param _homeChainId The chain ID where the home proxy is deployed.
+     */
+    function setHomeProxy(address _homeProxy, uint256 _homeChainId) external onlyGovernor {
+        require(homeProxy == address(0), "Home proxy already set");
+
+        homeProxy = _homeProxy;
+        homeChainId = _homeChainId;
+    }
+
+    /** @notice Changes the proportion of appeal fees that must be added to appeal cost when there is no winner or loser.
+     *  @param _sharedMultiplier The new shared multiplier value in basis points.
+     */
+    function changeSharedMultiplier(uint64 _sharedMultiplier) external onlyGovernor {
+        sharedMultiplier = _sharedMultiplier;
+    }
+
+    /** @notice Changes the proportion of appeal fees that must be added to appeal cost for the winning party.
+     *  @param _winnerMultiplier The new winner multiplier value in basis points.
+     */
+    function changeWinnerMultiplier(uint64 _winnerMultiplier) external onlyGovernor {
+        winnerMultiplier = _winnerMultiplier;
+    }
+
+    /** @notice Changes the proportion of appeal fees that must be added to appeal cost for the losing party.
+     *  @param _loserMultiplier The new loser multiplier value in basis points.
+     */
+    function changeLoserMultiplier(uint64 _loserMultiplier) external onlyGovernor {
+        loserMultiplier = _loserMultiplier;
+    }
+
+    /** @notice Updates the meta evidence used for disputes.
+     *  @param _metaEvidence URI to the new meta evidence file.
+     */
+    function changeMetaEvidence(string calldata _metaEvidence) external onlyGovernor {
+        require(initialized, "Metaevidence was not set.");
+        metaEvidenceUpdates++;
+        emit MetaEvidence(metaEvidenceUpdates, _metaEvidence);
+    }
+
+    /**
+     * @notice Requests arbitration for a given question ID.
+     * @dev Can be executed only if the contract has been initialized.
+     * @param _arbitrationID The ID of the arbitration, which is the ID of the related question converted into uint256.
+     * @param _contestedAnswer The answer the requester deems to be incorrect.
+     */
+    function requestArbitration(uint256 _arbitrationID, uint256 _contestedAnswer) external payable onlyIfInitialized {
```

Comment:
I just noticed that this method is not in the interface, but it should, as external code (Realitio interface) relies on it.

This should be the signature:
```solidity
function requestArbitration(bytes32 _questionID, bytes32 _contestedAnswer) external payable;
```